tests/osgi-tests/src/test/java/org/wso2/carbon/datasource/osgi/DataSourceServiceTest.java

```diff
@@ -47,7 +47,8 @@
                 mavenBundle().artifactId("HikariCP").groupId("com.zaxxer").version("2.4.1"),
                 mavenBundle().artifactId("h2").groupId("com.h2database").version("1.4.191"),
                 mavenBundle().artifactId("org.wso2.carbon.datasource.core").groupId("org.wso2.carbon.datasources")
-                        .versionAsInProject()
+                        .versionAsInProject(),
```

Comment:
Line break should have to be before the dot. That is the java 8 standard we have followed. Example[1].

[1] https://github.com/wso2/carbon-kernel/blob/master/core/src/main/java/org/wso2/carbon/kernel/internal/startupresolver/StartupOrderResolver.java